sass/styles.scss

```diff
@@ -0,0 +1,178 @@
+//Text-Settings//
+//quick question: can I have a variable with more than one css effect?//
+//I was wondering if I could combine/group codes and set it as a variable// 
+$sans: -apple-system, BlinkMacSystemFont, "Bebas Neue", cursive, sans-serif, Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+$serif: "Cormorant Garamond", serif; 
+$indent: 6vw;
+$text-small: 3vw;
+$text-large: 5vw;
+$undo: none;
+
+
+
+//Basic Document Settings//
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+    border: 0;
+}
+  
+body {
+    background-color: whitesmoke;
+    color: #242828;
+    font-family: $sans;
+    text-align: left;
+    font-weight: bold;
+    text-transform: uppercase;
+    line-height: 1;
+}
+
+h1, h2, h3 {
+    font-size: $text-large;
+  }
+  
+h4, p, span, ul {
+    font-size: $text-small;
+}
+      
+h4 {
+    font-style: italic;
+}
+
+ul {
+    list-style: none;
+    margin-top: 0;
+    margin-bottom: 0;
+    padding-left: 0;
+}
+  
+address {
+    font-style: normal;
+}
+
+span {
+    margin-left: $indent;
+    display: block;
+}
+  
+.u-sr-only {
+    border: 0;
+    clip: rect(0 0 0 0);
+    overflow: hidden;
+    margin: -1px;
+    padding: 0;
+    position: absolute;
+    height: 1px;
+    width: 1px;
+}
+  
+
+
+//Indented Em-dash settings//
+.em-dash-container {
+    display: flex; 
+    justify-content: flex-start; 
+    align-items: center;
+}
+  
+.em-dash {
+    background:black;
+    height: 0.25vw;
+    width: 5vw;
+    margin: 0 1vw 0 0; 
+}
+  
+
+
+//Text Formatting & Styling//
+.indent-event-description { 
+    margin-left: $indent;
+}
+  
+.large-font {
+    font-size: $text-large;
+}
+  
+.small-font {
+    font-size: $text-small;
+}
+  
+
+
+//Adjusting DIV items in Grids//
+main {
+    margin: 1.5%;
+    display: grid;
+    grid-template-columns: repeat(3, 32%);
+    grid-template-rows: repeat(10, 10%);
+    column-gap: 2%;
+    row-gap: 2%
+}
+  
+.item {
+    overflow-wrap: break-word;
+    hyphens: $undo;
+}
+  
+.item:nth-child(1) {
+    grid-column: 1/2;
+    grid-row: 1/3;
+    /*background: blue;*/
+}
+  
+.item:nth-child(2) {
+    grid-column: 1/2;
+    grid-row: 3/10;
+    /*background: red;*/
+}
+  
+.item:nth-child(3) {
+    grid-column: 2/3;
+    grid-row: 1/4;
+    /*background: purple;*/
+}
+
+.item:nth-child(4) {
+    grid-column: 2/3;
+    grid-row: 4/8;
+    /*background:greenyellow;*/
+}   
+        
+.item:nth-child(5) {
+    grid-column: 2/3;
+    grid-row: 8/9;
+    font-family: $serif;
+    text-transform: $undo;
```

Comment:
I would consider putting a class on the elements that get this specific font treatment.